examples/Demos/caduceusOculus.scn

```diff
@@ -1,3 +1,12 @@
+<!-- [ERROR]   [GLSLShader] ERROR: Compilation of Fragment shader failed:
+[ERROR]   [GLSLShader] From File: /home/arnaud/dev/sofa/build/install/share/sofa/shaders/phong.frag
+[ERROR]   [GLSLShader] 0:27(6): error: could not implicitly convert operands to relational operator
+0:27(6): error: LHS of `||' must be scalar boolean
+0:28(6): error: could not implicitly convert operands to relational operator
+0:28(6): error: LHS of `||' must be scalar boolean
+
+[ERROR]   [GLSLShader] SHADER compilation failed. -->
```

Comment:
hum... does it mean this error popup when you launch the scene?

Reply:
@epernod  Yes, couldn't solve it, shall I make an issue out of it?

Reply:
Done follow-up in #3530

Reply:
Thanks for creating the issue. I would now remove the comments you added in the scene